File: 2. Loop Problems/Loop problems in python/p4.py
# // Write a python program to find sum of all even numbers between 1 to n using loop

n = int(input("Enter n: "))
sum = 0
i = 1

while i<=n:
    if i%2==0 :
        sum = sum + i
    i= i +1
    
print(f"Sum= {sum}")
    
    

File: 2. Loop Problems/Loop Problems in c/p4.c
// Write a C program to find sum of all even numbers between 1 to n using loop

#include <stdio.h>

int main()
{
    int i = 1,sum=0,n;
    printf("Enter n: ");
    scanf("%d", &n);


    while (i <=n) {
        if (i%2==0) {
            sum = sum + i;  
        }
        i++;
    }

    printf("Sum= %d",sum);

}